### ISSUE TITLE
installation issue

### ISSUE THREAD

**Issue by feranto63** [theolind/pymysensors#287]
I'm using pymysensors since several months without problem or a Raspberry Pi 3.
Today I tried to update pymysensors but I got en error. See below logging info. Could you please help?
```
sudo pip3 install pymysensors
Downloading/unpacking pymysensors
  Downloading pymysensors-0.20.1-py3-none-any.whl (50kB): 50kB downloaded
Requirement already satisfied (use --upgrade to upgrade): crcmod>=1.7 in /usr/local/lib/python3.4/dist-packages/crcmod-1.7-py3.4-linux-armv7l.egg (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): click in /usr/local/lib/python3.4/dist-packages (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): IntelHex>=2.2.1 in /usr/local/lib/python3.4/dist-packages/intelhex-2.2.1-py3.4.egg (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): pyserial-asyncio>=0.4 in /usr/local/lib/python3.4/dist-packages/pyserial_asyncio-0.4-py3.4.egg (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): voluptuous>=0.11.1 in /usr/local/lib/python3.4/dist-packages (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): getmac in /usr/local/lib/python3.4/dist-packages (from pymysensors)
Requirement already satisfied (use --upgrade to upgrade): pyserial>=3.4 in /usr/local/lib/python3.4/dist-packages/pyserial-3.4-py3.4.egg (from pymysensors)
Installing collected packages: pymysensors
*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/__init__.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/__init__.py", line 226
    async def start(self):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/cli/gateway_mqtt.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/cli/gateway_mqtt.py", line 98
    async def async_start_mqtt_client(loop, broker, port):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_mqtt.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_mqtt.py", line 158
    async def get_gateway_id(self):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_serial.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_serial.py", line 79
    async def get_gateway_id(self):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_tcp.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/gateway_tcp.py", line 143
    async def get_gateway_id(self):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/task.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/task.py", line 173
    async def start(self):
            ^
SyntaxError: invalid syntax

*** Error compiling '/tmp/pip-build-3qngjiz5/pymysensors/mysensors/transport.py'...
  File "/tmp/pip-build-3qngjiz5/pymysensors/mysensors/transport.py", line 95
    async def connect(self):
            ^
SyntaxError: invalid syntax

Successfully installed pymysensors
Cleaning up...
```

**Comment by MartinHjelmare** [comment]
Looks like your Python version isn't 3.5 or higher. Please check the environment and Python version.

**Comment by feranto63** [comment]
solved installing python 3.6
